pilot/pkg/xds/monitoring.go

```diff
@@ -112,7 +112,7 @@ var (
 	pushContextInitTime = monitoring.NewDistribution(
 		"pilot_pushcontext_init_seconds",
 		"Total time in seconds Pilot takes to init pushContext.",
-		[]float64{.01, .1, 0.5, 1, 3, 5},
+		[]float64{.01, .1, 1, 3, 5, 10, 20, 30},
```

Comment:
Actually it exists. I think it might be common when there are a lot of CRs.

Reply:
It is not , if it is > 5s, we can imagine the pilot is hard to handle such load, the whole system is not reliable

Reply:
Thanks for your reply, but how to prove that 5s is an appropriate boundary value?

IMO, when it is > 5s, it will only affect the configuration sending time, and will not cause configuration sending errors. If I'm not critical about timing, then I don't care if it's 5s or longer.

Also, why is it appropriate to set the upper bound of [pushTime](https://github.com/istio/istio/blob/e34c12d1a9567c1b624472ef4c8a3997314fcef5/pilot/pkg/xds/monitoring.go#L121), [sendTime](https://github.com/istio/istio/blob/e34c12d1a9567c1b624472ef4c8a3997314fcef5/pilot/pkg/xds/monitoring.go#L128), [proxiesQueueTime](https://github.com/istio/istio/blob/e34c12d1a9567c1b624472ef4c8a3997314fcef5/pilot/pkg/xds/monitoring.go#L134)... to 30s ?

Reply:
It was set by experience.

Reply:
Hello @hzxuzhonghu @ramaraochavali, how much time do you generally spend in the production environment? thanks.